cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/UnmanagedClusterInventory.tsx

```diff
@@ -33,6 +33,14 @@ function UnmanagedClusterInventory() {
         setShowPageLoading(true);
         try {
             const data = await UnmanagedService.getUnmanagedClusters();
+            data.filter((cluster) => {
+                if (cluster.status === 'Unknown') {
+                    setNotification({
+                        status: NotificationStatus.DANGER,
+                        message: `Unmanaged Clusters ${cluster.name} is failed: Please Delete it first`,
```

Comment:
Small recommendation. Change to say `Unmanaged Clusters ${cluster.name} is failed: Please delete this cluster before attempting to create a new unmanaged cluster.`,

Reply:
Done

Reply:
Done